src/Makefile.am

```diff
@@ -372,13 +372,16 @@ libbitcoin_wallet_a_SOURCES = \
   wallet/rpcwallet.cpp \
   wallet/salvage.cpp \
   wallet/scriptpubkeyman.cpp \
-  wallet/sqlite.cpp \
   wallet/wallet.cpp \
   wallet/walletdb.cpp \
   wallet/walletutil.cpp \
   wallet/coinselection.cpp \
   $(BITCOIN_CORE_H)
 
+if USE_SQLITE
+libbitcoin_wallet_a_SOURCES += wallet/sqlite.cpp
```

Comment:
Shouldn't `wallet/sqlite.h` be included here too?

Reply:
Hmm. I would expect that systems without sqlite's `sqlite3.h` header would then fail to compile.

Reply:
It's not used despite being listed